sdcm/provision/network_configuration.py

```diff
@@ -89,6 +91,15 @@ def interface_ipv6_address(self):
         else:
             return None
 
+    @property
+    def device(self):
+        # Depend on network configuration:
+        # if broadcast_address.device_name if found
+        # else empty string.
+        if address_config := [conf for conf in self.scylla_network_config if conf["address"] == "broadcast_address"]:
+            return "".join([ni.device_name for ni in self.network_interfaces if ni.device_index == address_config[0]['nic']])
+        return ""
```

Comment:
maybe we should raise exception here ? 

it means this was called for unsupported backend, isn't it ?

Reply:
for now it's one nemesis, that can't work on any other backends.

so we are good, as long no one tries to use it and assume the wrong idea about it, in other places

Reply:
We use it for AWS only. This nemesis can be run with more then one network interface and it is supported in AWS only. So for now only AWS test will call this property